packages/web-app-files/src/components/SideBar/Shares/SpaceMembers.vue

```diff
@@ -2,7 +2,12 @@
   <div id="oc-files-sharing-sidebar" class="oc-position-relative">
     <oc-loader v-if="sharesLoading" :aria-label="$gettext('Loading members')" />
     <template v-else>
-      <invite-collaborator-form v-if="currentUserCanShare" key="new-collaborator" class="oc-my-s" />
+      <invite-collaborator-form
+        v-if="currentUserCanShare"
+        key="new-collaborator"
+        saving-button-title="Add"
```

Comment:
Needs to be wrapped in `$gettext('Add')`, otherwise it will not be extracted to transifex